packages/core/strapi/lib/commands/develop.js

```diff
@@ -131,6 +132,8 @@ function watchFileChanges({ dir, strapiInstance, watchIgnoreFiles, polling }) {
       '**/index.html',
       '**/public',
       '**/public/**',
+      strapiInstance.dirs.public,
+      urlJoin(strapiInstance.dirs.public, '**'),
```

Comment:
would path.join be enough here? that would avoid installing an extra dep?

Reply:
`path.join` would put `\` on windows platforms. I read that it doesn't work well on chokidar

![Capture d’écran 2022-03-02 à 15 39 31](https://user-images.githubusercontent.com/11708220/156383289-613db24d-32c9-4054-bf6a-e8ce6ab1a29f.png)

I can join by hand however if you want